### PR TITLE
Move and update editors page

### PR DESCRIPTION
**Pull request by grugnog**
Engineering is a better place for this - this isn't relevant to many on the team
I am also including IDEs, since that is a likely search term

----
:books: Documentation preview :books:: https://civicactions-handbook--1131.org.readthedocs.build/en/1131/

### CODE REVIEW

**Comment by grugnog** [comment]
[✖] https://supporters.eff.org/donate/ → Status: 404
 
 This looks like perhaps a temporary outage to me

**Comment by dmundra** [comment]
Thanks @grugnog. Did you want to also update the redirect? https://github.com/CivicActions/guidebook/blob/v2.1/.config/redirects.yml#L259-L261

File: .config/redirects.yml [hunk 1]
```diff
@@ -258,7 +258,11 @@
 
 - type: page
   from_url: /050-how-we-work/tools/text-editors/
-  to_url: /common-practices-tools/software-and-support/text-editors/
+  to_url: /practice-areas/engineering/text-editors-ides.md
+
+- type: page
+  from_url: /common-practices-tools/software-and-support/text-editors/
+  to_url: /practice-areas/engineering/text-editors-ides.md
```
Comment (dmundra) [inline .config/redirects.yml]: I don't think the redirects should have the `.md` path? It might not matter but for other pages we have not added it.

**Review by dmundra — APPROVED** [review]
Looking good. Thank you!

**Pull request merged**